ui/base/ime/input_method_minimal_unittest.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/base/ime/input_method_minimal.h"
```

Comment:
have you tried to do git cl format on this (input_method_minimal_unittest.cc) file? I guess it will move this include to be in alphabetical order as other includes as long as it's not a definition file of the InputMethodMinimal class.

Reply:
After I ran git cl format, it still exists at the top of the file. I think that is the reason why input_method_auralinux_unittest.cc also has xxx_auralinux.h at the top of it.

Reply:
ok!